### PR TITLE
This fixes a few links, embeds the html for the latest manpages, and includes IR-catala

### PR DESCRIPTION
**Pull request by AltGr**

### CODE REVIEW

File: src/pages/Doc.res
Comment (denismerigoux) [inline src/pages/Doc.res:1]: The web assets should be update to reflect the changes, I've filed https://github.com/CatalaLang/catala-web-assets/pull/2. Once it's merged and the `catala-web-assets` version bumped `package.json` should be updated too here.

**Comment by AltGr** [comment]
This includes changes that:
- add a link to https://gitlab.adullact.net/dgfip/ir-catala (and direct view of the corresponding pdf artifact)
- embed the view of live manpages generated by catala-ci (dled from https://catalalang.github.io/catala through js)
- fix the links to download the cheat-sheet (directed to the same artifacts page)
- API documentation is also now a simple link to the nightly builds at https://catalalang.github.io/catala/api-doc instead of being embedded
- the obsolete catala-legifrance manpage has been replaced by links to https://github.com/CatalaLang/catleg

Note that I used direct javascript code (through `%%raw`) to do the more advanced stuff (e.g. `fetch` the html manpages and integrate them in the layout) ; there probably are better ways but I just don't know enough about rescript to figure it out.

**Comment by AltGr** [comment]
Sneak peek

image

image

**Review by denismerigoux — APPROVED** [review]
Thank you !

File: src/pages/Doc.res [hunk 1]
```diff
@@ -4,31 +4,47 @@ open PageComponents
 
 module type ManPage = {
   let title: React.element
-  let html: string
+  let url: string
 }
 
+// I'd be happy to learn how to write this in plain rescript, please point to
+// the documentation
```
[the remark this answers is not in the document]
Comment (EmileRolley) [reply src/pages/Doc.res:11]: https://github.com/glennsl/rescript-fetch ?

File: src/pages/IRcatala.res [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  )
+}
+`)
+
+external fetch_unzip_and_show_pdf: (~url: string, ~filename: string, ~id: string) => unit =
+  "fetch_unzip_and_show_pdf"
+
+@react.component
+let make = () => {
+  let obj =
```
Comment (EmileRolley) [inline src/pages/IRcatala.res:51]: This is the first time I've seen an `<object>` tag, what's it for?

File: src/pages/IRcatala.res [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    <object
+      id="ir-catala-pdf"
+      style={ReactDOM.Style.make(
```
Comment (EmileRolley) [inline src/pages/IRcatala.res:54]: Couldn't this be done with tailwind instead?

File: src/pages/IRcatala.res (hunk 2 above)
Comment (AltGr) [reply src/pages/IRcatala.res:51]: (Multi)media objects in general, here it's used to embed the PDF (using the browser-native PDF viewer)
You can see this script in action at https://lgesbert.gitlabpages.inria.fr/catala/latest-pdf.html

File: src/pages/IRcatala.res (hunk 3 above)
Comment (AltGr) [reply src/pages/IRcatala.res:54]: Hm probably, I don't really know tailwind :grimacing:

File: src/pages/IRcatala.res [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    style=
+      ReactDOM.Style.make(
+        ~position="fixed",
+        ~left="0",
+        ~height="calc(100vh - 1.5rem)",
+        ~maxWidth="none",
+        ~width="100%",
+        ())
+  />
+  fetch_unzip_and_show_pdf(
```
[the remark this answers is not in the document]
Comment (AltGr) [reply src/pages/IRcatala.res:63]: I'd rather say it'a a horrible hack, but I must admit I like it :rofl:

**Pull request merged**